frontend/src/pages/projects/screens/detail/connections/ConnectionsDeleteModal.tsx

```diff
@@ -43,6 +62,47 @@ export const ConnectionsDeleteModal: React.FC<Props> = ({
     >
       The <b>{getDisplayNameFromK8sResource(deleteConnection)}</b> connection will be deleted, and
       its dependent resources will stop working.
+      {notebooksLoaded && !connectedNotebooks.length ? null : (
+        <div className="pf-v5-u-mt-md">
+          {!notebooksLoaded ? (
+            <Bullseye>
+              <Spinner size="md" />
+            </Bullseye>
+          ) : (
+            <>
+              <ExpandableSectionToggle
+                isExpanded={notebooksExpanded}
+                onToggle={setNotebooksExpanded}
+                contentId="expanded-connected-notebooks"
+                data-testid="connections-delete-notebooks-toggle"
+              >
+                <span>Workbenches </span>
+                <Badge isRead data-testid="connections-delete-notebooks-count">
+                  {connectedNotebooks.length}
+                </Badge>
+              </ExpandableSectionToggle>
+              <ExpandableSection
+                isExpanded={notebooksExpanded}
+                isDetached
+                contentId="expanded-connected-notebooks"
+              >
+                <TextContent>
+                  <TextList>
+                    {connectedNotebooks.map((notebook) => (
+                      <TextListItem
+                        key={notebook.metadata.name}
+                        data-testid="connections-delete-notebooks-item"
+                      >
+                        {getDisplayNameFromK8sResource(notebook)}
```

Comment:
[Looking at the mocks](https://www.figma.com/design/97akqulXiGIGtvxkdPLjMm/Data-connections-enhancement?node-id=2406-323841&t=IVjPOEBdHBNnscZr-4), I believe we need to append  " (Running)" to the display name, if `notebook.running === true`